packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.ts

```diff
@@ -406,6 +406,7 @@ export class HorizontalScroll extends FoundationElement {
 
         if (this.speed < 1) {
             this.scrollContainer.scrollLeft = newPosition;
+            this.$emit("scrollend");
```

Comment:
I do find it a little odd that the control is firing a `scrollend` event even when it doesn't consider itself scrolling (`this.scrolling === false`). My initial thought was that you could make `this.scrolling` observable and emit a scroll start/end event depending on the transition (`true` -> `false` === scrollend, `false -> true` === scrollstart) but that wouldn't work because scrolling is only set if speed > 1? I'm not familiar enough with this control to know why that is.

Anyway, if we're adding a "scrollend" event I think for parity we should probably emit a "scrollstart" event.

Reply:
I considered adding more events like scrollstart and scrollend but Ben sent the PR out for this. I figured, if it was needed, you could attach an onclick event handler to get the scrollstart, there was just no good way to get the scrollend.

Using this.scrolling makes a lot of sense though. It was initially just used to prevent scrolling from being initiated while it was currently scrolling. We'd just need to make sure that we set this.scrolling to true and then false during an instant scroll (speed=0).